docs/docsite/rst/user_guide/modules_support.rst

```diff
@@ -10,13 +10,13 @@ To help identify maintainers and understand how the included modules are officia
 Core
 ````
 
-:ref:`core_supported`. These are maintained by the Ansible Engineering Team.
+:ref:`Core<core_supported>` modules are maintained by the Ansible Engineering Team.
```

Comment:
I think we need to leave the word supported in here.  "Core module" implies importance whereas "Core supported" implies that the Core maintainers are responsible for it.  I don't want to get into discussions with users/customers/etc that a specific module is important enough to be a "Core module".

Reply:
Updated. Although the anchors are `*_supported`, the pages use the wording "* Maintained", so I made the labels that way.